src/components/Pursuance/PursuanceMenu.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Navbar, NavItem, Nav, ButtonGroup, Button, DropdownButton, MenuItem } from 'react-bootstrap';
+import { NavLink } from 'react-router-dom';
+import PursuanceMenuItem from './PursuanceMenuItem';
+import FaCheckSquareO from 'react-icons/lib/fa/check-square-o';
+import CommentsO from 'react-icons/lib/fa/comments-o';
+import PaperClip from 'react-icons/lib/fa/paperclip';
+import Group from 'react-icons/lib/fa/group';
+import Rocket from 'react-icons/lib/fa/rocket';
+import PlusCircle from 'react-icons/lib/fa/plus-circle';
+
+const PursuanceMenu = ({ pursuanceId }) => {
+  return (
+    <ButtonGroup vertical>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='New'
+          action='add'
+          icon={<PlusCircle size={28} />}
+        />
+      </Button>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='Tasks'
+          action='tasks'
+          icon={<FaCheckSquareO size={28} />}
+        />
+      </Button>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='Discuss'
+          action='discussion'
```

Comment:
@Spetastian How about calling both "[Dd]iscuss" just to be consistent?

Reply:
It would make more sens to call the action `discussions` plural, IMO. Since its an URL resourse that points to ongoing discussions. The label is just a human readable thing as in i want to "Discuss" this.

But i can change it if you dont agree on that.